core/arch/arm/kernel/boot.c

```diff
@@ -1165,12 +1171,18 @@ static bool cpu_nmfi_enabled(void)
  * the unpaged area.
  */
 void __weak boot_init_primary_late(unsigned long fdt,
-				   unsigned long tos_fw_config)
+				   unsigned long manifest_fdt)
```

Comment:
This local `manifest_fdt` and the global `manifest_dt` quite look alike, it a bit confusing.

Reply:
I'll think of something.